lua/pac3/core/server/util.lua

```diff
@@ -51,11 +51,7 @@ function pac.RatelimitAlert( ply, id, message )
 		ply.pac_ratelimit_alerts = {}
 	end
 
-	if not ply.pac_ratelimit_alerts[id] then
-		ply.pac_ratelimit_alerts[id] = CurTime() + 3
-	end
-
-	if CurTime() > ply.pac_ratelimit_alerts[id] then
+	if CurTime() >= ply.pac_ratelimit_alerts[id] or 0 then
```

Comment:
I'm unsure about operator precedence here, wouldn't this error?

(CurTime() >= nil) or (0)
vs
(CurTime()) >= (nil or 0)

Reply:
Yep, good catch - just pushed a fix 👍

Reply:
Yeah that would've errored

![image](https://user-images.githubusercontent.com/69946827/203032389-57e4ccb1-b9bc-49fa-ac99-d4eed1f7a56e.png)

New commit fixes it

![image](https://user-images.githubusercontent.com/69946827/203032420-58827a05-4361-4daf-9d17-69ced2e97ef2.png)